backend/app/services/mcp_providers/service.py

```
@@ -0,0 +1,188 @@
+                    "unauthorized",
+                )
+            elif error_code == "server_error":
+                return (
+                    False,
+                    "Provider server error. Please try again later.",
+                    [],
+                    "server_error",
+                )
+            else:
+                return False, "Failed to sync servers", [], error_code
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)